test/point_cloud.jl

```diff
@@ -0,0 +1,29 @@
+using Random
+
+# Generate a rectangular point cloud, optionally with a perturbation in the point positions
+function point_cloud(n_points_per_dimension;
+                     seed = 1, perturbation_factor_position = 1.0)
```

Comment:
Why is this space between ` = `  not detected by the JuliaFormatter?

Reply:
I use a different JuliaFormatter configuration here without `whitespace_in_kwargs = false`. This is consistent with Trixi's new style and SciML, and I thought about changing it for TrixiParticles.jl as well in the future.